### PR TITLE
Packaging changes

### PR DESCRIPTION
**Pull request by michaelfig**
Minor details to get `npx agoric@0.9.0` to run again.

### CODE REVIEW

File: packages/zoe/package.json [hunk 1]
```diff
@@ -61,6 +61,7 @@
   },
   "files": [
     "src/",
+    "contractFacet.js",
```
Comment (katelynsills) [inline packages/zoe/package.json:64]: Hmm, this is already in src/

Comment (michaelfig) [reply packages/zoe/package.json:64]: There's one in the package root, too.  That's for external clients to use `@agoric/zoe/contractFacet`.

Comment (katelynsills) [reply packages/zoe/package.json:64]: Oh, I see, thanks!

**Review by katelynsills — APPROVED** [review]
Looks fine to me

**Pull request merged**